### DISCUSSION TITLE
无法正常运行，日志如下

### DISCUSSION THREAD

**Discussion by azsummer** [messense/aliyundrive-webdav#149] [Q&A] [answered]
### 问题描述

WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out

### 版本

aliyundrive-webdav 0.4.0 

### 运行平台

斐讯N1

### 日志

Error: initialize aliyundrive file system failed
Nov 14 15:24:30.294  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:24:41.297  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:24:52.305  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:25:03.309  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:25:14.313  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:25:25.315  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:25:36.321  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:25:47.326  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:25:58.333  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:26:09.338  WARN aliyundrive_webdav::drive: refresh token failed, will wait and try error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Nov 14 15:26:10.339 ERROR aliyundrive_webdav::drive: refresh token failed: error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: operation timed out
Error: initialize aliyundrive file system failed

**Comment by messense** [comment, marked as answer]
> error trying to connect: operation timed out

网络问题，检查能不能正常访问 https://websv.aliyundrive.com/token/refresh

```bash
curl -v https://websv.aliyundrive.com/token/refresh
```

**Reply by messense** [reply]
BTW, 可以先升级到最新版本。

**Reply by azsummer** [reply]
尝试了下，好像无法访问，宽带是移动的，我使用新加坡的服务器尝试可访问

**Reply by tianqu36** [reply]
同样的问题，电信宽带无法访问,同样境外的服务器可以访问,可能流量太大ip被禁用？

请问author，数据会通过这个地址吗？ 这个域名我才用代理服务器的模式，请问这样是否会大量占用我的服务器流量？

**Reply by messense** [reply]
`websv.aliyundrive.com` 这个域名只会用来刷新 token，其他流量不会走这个域名。

**Reply by tianqu36** [reply]
> `websv.aliyundrive.com` 这个域名只会用来刷新 token，其他流量不会走这个域名。

OK，那就放心把这个域名走代理了。感谢！
